src/gui/widgets/Knob.cpp

```diff
@@ -497,8 +499,8 @@ float Knob::getValue( const QPoint & _p )
 {
 	float value;
 
-	// arcane mathemagicks for calculating knob movement
-	value = ( ( _p.y() + _p.y() * qMin( qAbs( _p.y() / 2.5f ), 6.0f ) ) ) / 12.0f;
+	// knob value increase is linear to mouse movement
+	value = .09f * _p.y();
```

Comment:
On Mac using the trackpad, a `0.4x` multiplier is pretty good.  Will test on Ubuntu 20.04 next.

```cpp
value = .4f * _p.y();
```

Reply:
Fedora 30 VM, `.4f` works well for me!

Reply:
On Ubuntu 20.04 running in a VM, the existing `.09x` multiplier seems just fine.  Will continue investigating why there's a disparity.

Reply:
Forcing `SetPosStatusType::NoEffect` on Ubuntu 20.04 reproduces the sensitivity issue with macOS.  My theory is that this sensitivity is a result of the `SetPosStatusType::Works` logic.  Will post in the main thread.